common/ai_strategies/00_default_strategy.txt

```diff
@@ -809,6 +809,18 @@ ai_strategy_default = {
 			add = 100
 		}
 		
+		#Please stop killing your vassal burgundy
+		if = {
```

Comment:
Maybe it is better to add something in secret goal to make them more likely to be protective?

Reply:
Will do in future, rn the event for them to exist doesn't trigger yet